nativeauthenticator/handlers.py

```diff
@@ -188,35 +224,45 @@ async def get(self):
         self.finish(html)
 
 
-class ChangeAuthorizationHandler(LocalBase):
+class ToggleAuthorizationHandler(LocalBase):
+    """Responsible for the authorize/[someusername] page,
+    which immediately redirects after toggling the
+    respective user's authorization status."""
```

Comment:
Off topic to the PR, just a comment about the logic:
In general, I think this is a risky approach. I'd prefer to have a "set to true" / "set to false" kind of handler rather than toggling.

Reply:
I have opened #184 for this.